content/src/service/access/AccessCheckerForWearables.ts

```diff
@@ -100,6 +152,19 @@ export class AccessCheckerForWearables {
     }
   }
 
+  private async calculateHashes(content?: Map<string, string>, metadata?: any) {
+    // Compare both by key and hash
+    const compare = (a: { key: string; hash: string }, b: { key: string; hash: string }) => {
+      if (a.hash > b.hash) return 1
+      else if (a.hash < b.hash) return -1
+      else return a.key > b.key ? 1 : -1
+    }
+    const entries = Array.from(content?.entries() ?? [])
+    const contentAsJson = entries.map(([key, hash]) => ({ key, hash })).sort(compare)
+    const buffer = Buffer.from(JSON.stringify({ content: contentAsJson, metadata }))
+    return Promise.all([Hashing.calculateBufferHash(buffer), Hashing.calculateIPFSHash(buffer)])
+  }
+
```

Comment:
I believe this function should be shared in `catalyst-common`, otherwise the implementation of the hash may vary from what is actually being calculated from dApps. We could remove the code from here in another PR, I will send the PR to commons now: https://github.com/decentraland/catalyst-commons/pull/127